docs/react-testing-library/api.mdx

```diff
@@ -333,9 +333,15 @@ component results in more readable and robust tests since the thing you want to
 test is not hidden behind an abstraction.
 
 ```typescript
-function renderHook<Result, Props>(
-  render: (props: Props) => Result,
-  options?: RenderHookOptions<Props>,
+function renderHook<
+  Result,
+  Props,
+  Q extends Queries = typeof queries,
+  Container extends Element | DocumentFragment = HTMLElement,
+  BaseElement extends Element | DocumentFragment = Container,
+>(
+  render: (initialProps: Props) => Result,
+  options?: RenderHookOptions<Props, Q, Container, BaseElement>,
 ): RenderHookResult<Result, Props>
```

Comment:
https://github.com/testing-library/react-testing-library/blob/main/types/index.d.ts#L140

Modified the type to match the actual type.